src/towncrier/test/test_format.py

```diff
@@ -327,6 +331,38 @@ def test_line_wrapping_disabled(self):
         output = render_fragments(
             template,
             None,
+            "",
+            fragments,
+            definitions,
+            ["-", "~"],
+            wrap=False,
+            versiondata={"name": "MyProject", "version": "1.0", "date": "never"},
+        )
+        self.assertEqual(output, expected_output)
+
+    def test_title_format_specified(self):
+        """
+        Title format replaces default top line rendering.
+        """
+        self.maxDiff = None
+
+        fragments = {}
+
+        definitions = OrderedDict()
+
+        expected_output = u"""A custom top line
+=================
+"""  # NOQA
```

Comment:
minor comment. Why do we need no qa here?

Reply:
I'm guessing it was the indentation?  Switched to `dedent()`.